lib/country-bounding-boxes.ts

```diff
@@ -37,6 +37,7 @@ const countryBoundingBoxData: Record<string, CountryBoundingBox | undefined> = {
   Cameroon: [8.49, 1.73, 16.01, 12.86],
   "Democratic Republic of the Congo": [12.18, -13.26, 31.17, 5.26],
   "Republic of Congo": [11.09, -5.04, 18.45, 3.73],
+  "Congo": [11.09, -5.04, 18.45, 3.73],
```

Comment:
Just a quick band-aid because some of the countries were renamed in airtable so the country zoom was broken for those countries. Yes I know it's silly that I tied the name in airtable to the bounding box, I've got an idea that might (?) fix this eventually but you'll have to bear with this for now, sorry :bow: